### ISSUE TITLE
Transalation API

### ISSUE THREAD

**Issue by shantanuo** [jalajthanaki/POS-tag-workshop#2]
Is it possible to use spacy to translate english text to hindi?

**Comment by jalajthanaki** [comment]
Yes. Spacy can be used. It's majorly depend on what kind of features you are trying to extract using Spacy.  Spacy can be useful if you are trying to incorporate following features.
* POS tags for English Language
* Spacy dependency parser is also useful
* You can use NER module as well

If you want to build the translation API using Deep Learning then I would suggest please refer [this repository](https://github.com/jalajthanaki/NLPython/tree/master/ch9/MT/Machine_Translation_GR_EN)

**Comment by shantanuo** [comment]
Can you show sample code that will translate english text to Hindi using spacy?

**Comment by jalajthanaki** [comment]
Shantanuo, As per my knowledge, Spacy does not have direct translation API for EN-HN. In that case you need to build the translation system by your own or you can use Google's translation APIs. [Here is the link](https://cloud.google.com/translate/docs/)

If you are want to use spacy then you can extract the features such as POS tag, Dependency relations among phrases, NER and so on. After that you need to apply Statistical Machine Translation techniques but as per my experience, Deep Learning (DL) works better so I have suggested that to you.

**Comment by shantanuo** [comment]
I am aware of Google / Microsoft / Amazon API. But I am looking for something from open source and /or Machine Learned model. I had seen an example on the net. I guess it used spacy to translate English to Hindi. But I do not remember the link of blog post.
If you come across spacy (or any other) api, then please me know.

**Comment by jalajthanaki** [comment]
Sure, I will.

**Comment by jalajthanaki** [comment]
Hi Shantanuo,
[Yandex Translation API](https://tech.yandex.com/translate/doc/dg/concepts/api-overview-docpage/): This is the service which provides the EN-HN translation. It is free tool but it has limit on text content and no. of request in time limit. You can take a look.
